library/cipher.c

```diff
@@ -1236,7 +1236,7 @@ int mbedtls_cipher_crypt( mbedtls_cipher_context_t *ctx,
             (mbedtls_cipher_context_psa *) ctx->cipher_ctx;
 
         psa_status_t status;
-        psa_cipher_operation_t cipher_op;
+        psa_cipher_operation_t cipher_op = PSA_CIPHER_OPERATION_INIT;
```

Comment:
Why wasn't this caught by CI? Are we missing a test case or a test configuration?

Reply:
This was caught by CI, but it wasn't a failure until we started checking that the context is initialized before allowing setup() (by fixing #10). Same with the X.509 issues. all.sh catches this when it runs "make test" when using the `MBEDTLS_USE_PSA_CRYPTO` option.